src/boot_O2_g3/boot_main.c

```diff
@@ -1,6 +1,7 @@
 #include "prevent_bss_reordering.h"
 #include "global.h"
 #include "stack.h"
+#include "stackcheck.h"
```

Comment:
Just below here could you remove the word 'Thread' from the two stack variables. None of the other stack variables have it.